src/Versions.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+    <PropertyGroup>
```

Comment:
please lets start using [Central Package Management](https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management) pattern for our new projects.